editor.planx.uk/src/pages/FlowEditor/components/EditorMenu.tsx

```diff
@@ -68,10 +69,10 @@ const MenuButton = styled(IconButton, {
 
 function EditorMenu() {
   const { navigate } = useNavigation();
-  const { lastChunk } = useCurrentRoute();
+  const { url } = useCurrentRoute();
   const rootPath = rootFlowPath();
 
-  const isActive = (route: string) => lastChunk.url.pathname.endsWith(route);
+  const isActive = (route: string) => url.pathname.endsWith(route);
```

Comment:
This change was required as the `/feedback` route (in its own file) ended with a `/`